partner_identification/views/res_partner_view.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <!-- Modification of Partner - Adding Tab for Idenification Numbers -->
+    <record model="ir.ui.view" id="view_partner_form">
+        <field name="name">res.partner.form.id_number</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="priority" eval="10" />
+        <field name="arch" type="xml">
+            <page name='sales_purchases' position="after">
+                <page string="ID Numbers" name="id_numbers">
+                    <field
+                        name="id_numbers"
+                        colspan="4"
+                        nolabel="1"
+                        widget="one2many_list"
+                        context="{'default_partner_id': active_id}"
```

Comment:
This indicate to have default partner when we add a line, but it doesn't show up when I test it.

Reply:
![image](https://user-images.githubusercontent.com/1973598/107839393-2d977500-6dde-11eb-8a55-7c8ef6a6282e.png)

Reply:
@kittiu Thanks for your feedbacks, fixed in https://github.com/OCA/partner-contact/pull/1058/commits/2d8aef4bf5cd1c766b1b7c16a845b93e9dd83976